nemo/collections/tts/helpers/helpers.py

```diff
@@ -547,12 +547,11 @@ def regulate_len(
     reps = durations.float() / pace
     reps = (reps + 0.5).floor().long()
     dec_lens = reps.sum(dim=1)
-
     if replicate_to_nearest_multiple:
```

Comment:
Can we test this function?

Reply:
Well I did quite a bit of ad hoc testing - what exactly do you have in mind ?

Reply:
A small unit test in https://github.com/NVIDIA/NeMo/tree/main/tests/collections/tts that tests input of three size: [group_size-1, group_size, group_size+1]

Reply:
Well, perhaps that test should have been added when replicate was first introduced (with a bug :) ). 
Anyway, you got it: added test_helpers.py